tests/driver/CMakeLists.txt

```diff
@@ -0,0 +1,87 @@
+if (BUILD_ACQUIRE_DRIVER_ZARR)
```

Comment:
This relates to the previous comment. If you add this subdirectory from the driver's CMake file (which is already added based on the same condition), you could remove this if-statement.